File: Mudanças/MIssão com Ruby/Questão 1.rb

input =[
    { "name"=> "Maria Neusa de Aparecida", "cpf"=> "97905796671",
    "state"=> "Sao Paulo", "value"=> "1234" },
    { "name": "Ricardo Fontes", "cpf": "44010762900",
    "state": "Rio Grande do Sul", "value": "567" }
]

def solucao(nome)
    s =nome.size 
    for k in 0..s -1 do 
        nome[k].each do |chave, valor|
            if chave != "value"
                print valor[0,11]
            else
                print ("\s") + valor[0,11]
                puts 
            end
        end
    end
end

solucao(input)

File: Teste com python/Questão1.py

import yaml


input = [
{ "name": "Maria Neusa de Aparecida", "cpf": "97905796671",
"state": "Sao Paulo", "value": "1234" },
{ "name": "Ricardo Fontes", "cpf": "44010762900",
"state": "Rio Grande do Sul", "value": "567" }
]
def solucao(arg):
# Retornar um string
    for Chave, item in arg[0].items():
        if Chave=="value":
            print(f" {item[:11]}",end="")
            
        else:
            print(f"{str(item[:11])}",end="") 
    print("\n")
    for Chave, item in arg[1].items() :
        if Chave=="value":
            print(f"{item[:11]}",end="")
        
        else:
            print(f"{str(item[:11])}",end="")
                
pass

solucao(input) == "Maria Neusa97905796671Sao Paulo 1234 \nRicardo Fon44010762900Rio Grande 567"


